asknicely/mysql-lambda
# WEB-12610: Update for MySQL 8

# Key change

Have `make mysql` produce a file containing MySQL 8 files instead of MySQL 5.7 files.

# How this was done

Merged in the latest changes from megastep:master.

# Also in this PR

We previously deleted the code related to other database systems like postgresql.  This complicated the merge, and is likely to complicated future merges, so I've restored the deleted files (and merged in their latest changes).

Locked Dockerfile.mysql to pull specifically from version 2 of amazonlinux (see https://github.com/megastep/mysql-lambda/issues/6)

Configured `zip` to preserve symlinks, to save disk space once expanded.

Updated our readme.

# To test

Run `make mysql` and confirm you get a zip file.  Examine the contents of the zip file and make sure they seem reasonable.

Put that zip file in asknicelydo, deploy to staging, and confirm things that use mysql, mysqldump, zip, and unzip are working.  (I believe this can be done later after approving this PR.)
